.circleci/config.yml

```diff
@@ -25,58 +25,8 @@ jobs:
             yarn ci-check
             yarn lerna run ci-check
 
-  artifacts:
-    docker:
-      - image: circleci/node:10.15-browsers
-    working_directory: ~/repo
-    steps:
-      - checkout
-      - run:
-          name: Install yarn
-          command: |
-            curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.16.0
```

Comment:
also curious why we don't have to install yarn in github actions?

Reply:
We check in our Yarn versions (under `.yarn/releases`) and so as long as the docker image has yarn it should redirect to the latest version 😄